runtime/integration-tests/Cargo.toml

```diff
@@ -9,88 +9,89 @@ repository = "https://github.com/centrifuge/centrifuge-chain"
 
 
 [dependencies]
-fudge = { git = "https://github.com/centrifuge/fudge", branch = "polkadot-v0.9.26"}
+fudge = { git = "https://github.com/centrifuge/fudge", branch = "polkadot-v0.9.29" }
 tokio = { version = "1.15", features = ["macros"] }
-tracing = "0.1"
+tracing = "0.1.36"
```

Comment:
Why did we need to bump this? Compatability with Fudge or Substrate?

Reply:
Reverted! Must have slipped in during updating dependencies!